spec/models/user_spec.rb

```diff
@@ -297,4 +297,25 @@
       expect { do_purge }.to change(User, :count).by(0)
     end
   end
+
+  #sam Unit test
+  describe User do
+    let(:emails) {  ['a@a.com','b@b.com'] }
+    let(:users) { double('Users') }
+        it '.superadmin_emails' do 
+          expect(users). to receive(:pluck).with(:email).and_return(emails)
+          expect(User).to receive(:superadmins).and_return(users)
+          expect(User.superadmin_emails).to eq emails
+        end
+  end
+
+  #integration test 
```

Comment:
placeholder fine - don't remove the unit test.

I can't really follow the logic of the superadmin account creation in this context.  For me the unit test is stand alone and completely unrelated to the functionality for generating emails for the superadmin.  Just even trying to think about them here gets me confused :-)  I don't think that can help us do this PR ...

Do you want to set up a pairing session to get to the bottom of things?